build.gradle

```diff
@@ -167,8 +167,12 @@ dependencies {
 
     integrationTestImplementation "org.testcontainers:junit-jupiter:$testcontainersVersion"
     integrationTestImplementation "org.testcontainers:kafka:$testcontainersVersion" // this is not Kafka version
-    integrationTestImplementation "io.confluent:kafka-avro-serializer:$confluentPlatformVersion"
-    integrationTestImplementation "io.confluent:kafka-connect-avro-converter:$confluentPlatformVersion"
+    integrationTestImplementation("io.confluent:kafka-avro-serializer:$confluentPlatformVersion") {
+        exclude group: "org.apache.kafka", module: "kafka-clients"
+    }
+    integrationTestImplementation("io.confluent:kafka-connect-avro-converter:$confluentPlatformVersion") {
+        exclude group: "org.apache.kafka", module: "kafka-clients"
+    }
```

Comment:
This is needed to prevent using the newer version of `kafka-clients` in the integration tests, which breaks them.